contingency/contingency-dsl/src/test/groovy/com/powsybl/contingency/dsl/ProbabilityContingencyDslExtension.groovy

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+package com.powsybl.contingency.dsl
+
+import com.google.auto.service.AutoService
+import com.powsybl.commons.extensions.Extension
+import com.powsybl.contingency.Contingency
+
+/**
+ * @author Paul Bui-Quang <paul.buiquang at rte-france.com>
+ */
+@AutoService(ExtendableDslExtension.class)
+class ProbabilityContingencyDslExtension implements ExtendableDslExtension<Contingency>{
```

Comment:
Coding style: a space is missing at the end of the line, just before the opening brace.